site.js

```diff
@@ -6,81 +6,35 @@ var timeSort = function (l, r) {
     return r.created_at.localeCompare(l.created_at);
 };
 
-var repoDefaults = [
-    { repo: "highfive", language: "L-python" },
-    { repo: "servo-starters", language: "L-javascript" },
-    { repo: "crowbot", language: "L-javascript" },
-    { repo: "saltfs", language: "L-saltstack" },
-];
-
 var langLabels = [{
-        name: "L-rust",
-        color: "bfd4f2",
-        url:  "https://api.github.com/servo/servo/labels/L-rust",
-        selected: true
-    },
-    {
-        name: "L-python",
-        color: "bfd4f2",
-        url:  "https://api.github.com/servo/servo/labels/L-python",
-        selected: true
-    },
-    {
-        name: "L-javascript",
-        color: "bfd4f2",
-        url:  "https://api.github.com/servo/servo/labels/L-javascript",
-        selected: true
-    },
-    {
-        name: "L-saltstack",
-        color: "bfd4f2",
-        url:  "https://api.github.com/servo/servo/labels/L-saltstack",
-        selected: true
-    },
-];
+    name: 'lang-python',
+    color: 'bfd4f2',
+    url: 'https://api.github.com/servo/servo/labels/L-python',
```

Comment:
This should be updated to anidata, but I'm not sure which repo should be used as the "reference" repo yet.